api/hiveextension/v1beta1/agentclusterinstall_types.go

```diff
@@ -171,6 +171,10 @@ type AgentClusterInstallSpec struct {
 	// Proxy defines the proxy settings used for the install config
 	// +optional
 	Proxy *Proxy `json:"proxy,omitempty"`
+
+	// PlatformType is the name for the specific platform upon which to perform the installation.
+	// +optional
+	PlatformType PlatformType `json:"platformType,omitempty"`
```

Comment:
Can you change the object to something like 'PlatformTypeEnum'? Seems ACI isn't populating PlatformType correctly when using the same name for both.

Reply:
It is working on the CI and on my test machine. I wish to leave it like this because this is the naming convention of Openshift.